dist/gaia-header.js

```diff
@@ -900,6 +902,9 @@ module.exports = component.register('gaia-header', {
       subtree: true
     });
 
+    window.addEventListener('resize', this);
+    this._resizeHandled = false;
```

Comment:
Why is this inside the mutation observer functions. Why can't we listen/unlisten on attached/detached?